packages/tldraw/src/lib/ui/context/actions.tsx

```diff
@@ -1478,6 +1481,66 @@ export function ActionsProvider({ overrides, children }: ActionsProviderProps) {
 					trackEvent('change-page', { source, direction: 'next' })
 				},
 			},
+			{
```

Comment:
not 100%, but i feel like these should be part of the select tool rather than part of the UI. I don't think i would expect disabling the UI to also disable keyboard navigation. wdyt?

Reply:
ok, yeah i think this is the better call 👍